torch/_tensor_docs.py

```diff
@@ -1741,25 +1767,24 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('sparse_mask',
                r"""
-sparse_mask(input, mask) -> Tensor
+sparse_mask(mask) -> Tensor
```

Comment:
what happens when this is called on a non-sparse tensor?

Reply:
Actually, `sparse_mask` must be called on a strided tensor. Calling on a sparse tensor, one currently gets
```
RuntimeError: Cannot access data pointer of Tensor that doesn't have storage
```

The updated docs mention that `self` must be a strided tensor.